src/components/tables/EntityTable/TableHeader.tsx

```diff
@@ -107,7 +103,9 @@ function EntityTableHeader({
                             }
                             sx={tableCellSX}
                         >
-                            {selectData && column.field && !hide ? (
+                            {column.renderInlineHeader && !hide ? (
+                                column.renderInlineHeader(index)
+                            ) : selectData && column.field && !hide ? (
```

Comment:
Need to add function that can handle rendering within the default `TableCell` tags